stderr.d.ts

```diff
@@ -0,0 +1,6 @@
+import Chalk from '.';
+
+declare const chalk: Chalk;
+
+export default chalk;
```

Comment:
Does `export default chalk from '.';` not work?

Reply:
Oh, I really don't know if that construct is valid

Reply:
No, your suggestion does not work.
If you wanna play with it you can :smiling_imp: